scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: This function is vulnerable to command injection attacks[^1]. User input is directly concatenated into a shell command without proper sanitization.

Recommendation: Avoid using `child_process.exec` with user input. If shell execution is necessary, use `child_process.execFile` or `child_process.spawn` with proper input validation and sanitization.
```suggestion
function executeCommand(userInput: string) {
    // Use execFile instead of exec, and pass arguments separately
    child_process.execFile('ls', [userInput], (error, stdout, stderr) => {
        if (error) {
            console.error(`Error: ${error.message}`);
            return;
        }
        console.log(stdout);
    });
}
```

[^1]: CWE-78: Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection') - https://cwe.mitre.org/data/definitions/78.html